recipes/aldjemy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "aldjemy" %}
+{% set version = "0.6.0"%}
+{% set sha256 = "a0f8e684d5c7b97dff64d0da0d75ca6a1f3062d6eca9e54c188e7babbe2439fa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sqlalchemy >0.7.1
+
+  run:
+    - python
+    - sqlalchemy >0.7.1
+
+test:
+  imports:
+    - aldjemy
+
+  commands:
+    # django 1.11 yields
+    # ImportError: No module named test1.urls 
+    - 'cd test_project; python manage.py test'  # [not win]
+
+  requires:
+    - django <1.11
+    - django-extensions
+    - sqlalchemy >0.7.1
+
+  source_files:
+    - test_project
+
+about:
+  home: https://github.com/Deepwalker/aldjemy/
+  license: BSD
```

Comment:
Could you raise an issue on the source repo asking them to add a license file and include it in the `MANIFEST.in` and then copy a link to that issue here?